core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/different/DifferentOpDesc.scala

```diff
@@ -0,0 +1,32 @@
+package edu.uci.ics.texera.workflow.operators.different
+
+import com.google.common.base.Preconditions
+import edu.uci.ics.amber.engine.operators.OpExecConfig
+import edu.uci.ics.texera.workflow.common.metadata.{
+  InputPort,
+  OperatorGroupConstants,
+  OperatorInfo,
+  OutputPort
+}
+import edu.uci.ics.texera.workflow.common.operators.OperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{OperatorSchemaInfo, Schema}
+
+class DifferentOpDesc extends OperatorDescriptor {
+  override def operatorExecutor(operatorSchemaInfo: OperatorSchemaInfo): OpExecConfig = {
+    new DifferentOpExecConf(operatorIdentifier)
+  }
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "Different",
+      "Find the set different of two inputs",
+      OperatorGroupConstants.UTILITY_GROUP,
+      inputPorts = List(InputPort()),
```

Comment:
this doesn't look right to me, this only specifies one input port. Furthermore, for this difference, I assume you are calculating `left - right`, so left and right matters. It's good to also specify a port name, check the hash join op desc.

Reply:
Already modified.